src/extension/index.ts

```diff
@@ -164,203 +148,93 @@ export async function activate(context: vscode.ExtensionContext) {
     )
   );
 
-  registerCommand(context, "neo3-visual-devtracker.neo.newContract", () =>
-    Templates.newContract(context)
-  );
+  registerCommand(context, "neo3-visual-devtracker.neo.newContract", () => Templates.newContract(context));
```

Comment:
Note, I *don't* like the old formatting where the `() =>` was on one line and the call was on the next, so I'm cool with this change or putting the entire lambda on the next line

Reply:
Same as before, this is the result of auto-formatting. As we have wider screens now the single-line version looks better imo.